app/util/IpmiCommand.scala

```diff
@@ -72,7 +72,7 @@ abstract class IpmiCommand extends BackgroundProcess[Option[CommandResult]] {
     Some(cr)
   }
 
-  override protected def defaultTimeout: Duration = Duration(2, TimeUnit.MILLISECONDS)
```

Comment:
Really? This seems like a crazy small default timeout. Not related to this change, but...